doc/cookbook/source/examples/classifier/lmnn.rst

```diff
@@ -0,0 +1,47 @@
+=================================
+Largest Margin Nearest Neighbours
```

Comment:
please dont abbreviate filenames. Call it "largest_margin_nearest_neighbours.rst"